druid/src/widget/flex.rs

```diff
@@ -338,8 +515,8 @@ impl<T: Data> Widget<T> for Flex<T> {
         for child in &mut self.children {
             if child.params.flex != 0.0 {
                 let major = remaining * child.params.flex / flex_sum;
+                let min_major = 0.0;
 
-                let min_major = if major.is_infinite() { 0.0 } else { major };
```

Comment:
This feels like a behavior change, but I'm not quite tracking.

Reply:
it is; we used to always pass tight constraints to flex children, and now we always pass loose constraints.